d2l-hypermedia-constants.js

```diff
@@ -0,0 +1,15 @@
+import {Actions, Classes, Rels} from 'index.js';
+
+window.D2L = window.D2L || {};
+window.D2L.Hypermedia = window.D2L.Hypermedia || {};
+window.D2L.Hypermedia.Constants = {
+	Actions: Actions,
+	Classes: Classes,
+	Rels: Rels
+};
+// for backwards-compatibility
+window.D2L.Hypermedia.HMConstantsBehavior = {
```

Comment:
Since this is going to be a breaking change anyway, is there a need to keep this around?

Reply:
To make the conversion easier to Polymer 3, you could name this file d2l-hm-constants-behavior.js instead?

Reply:
Is this still going to be converted? [This comment](https://github.com/Brightspace/d2l-hypermedia-constants/pull/76#issuecomment-437502642) from the other PR gave me the impression that this won't live as a Polymer component anymore at all, although I could be wrong!

Reply:
Yeah the intention here is that it's no longer a Polymer behaviour at all... and never really was since it just installed stuff on a global variable.